src/System.Private.Xml/src/System/Xml/Serialization/XmlRootAttribute.cs

```diff
@@ -88,9 +88,19 @@ internal bool IsNullableSpecified
             get { return _nullableSpecified; }
         }
 
+        internal bool GetIsNullableSpecified()
+        {
+            return this.IsNullableSpecified;
+        }
+
         internal string Key
         {
             get { return (_ns == null ? String.Empty : _ns) + ":" + ElementName + ":" + _nullable.ToString(); }
         }
+
+        internal string GetKey()
```

Comment:
It is to make the code be consistent in both xml and codegen project so we don't have to add if else condition.